libmscore/xml.cpp

```diff
@@ -625,6 +625,36 @@ QString Xml::xmlString(const QString& s)
       }
 
 //---------------------------------------------------------
+//   tagEscape
+//---------------------------------------------------------
+
+QString Xml::tagEscape(QString s, QString tag)
+      {
+      QString openTag = "<" + tag + ">";
+      QString openProxy = "!!" + tag + "!!";
+      QString closeTag = "</" + tag + ">";
+      QString closeProxy = "!!/" + tag + "!!";
+      s.replace(openTag, openProxy);
+      s.replace(closeTag, closeProxy);
+      s = Xml::xmlString(s);
+      s.replace(openProxy, openTag);
+      s.replace(closeProxy, closeTag);
+      return s;
+      }
+
+//---------------------------------------------------------
+//   unEscape
+//---------------------------------------------------------
+
+QString Xml::unEscape(QString s)
```

Comment:
Since Xml::xmlString is only creating entities for <, >, &, and ", I think that's all we need to unescape as well.  Other characters like the copyright symbol are safe to leave raw.  So I'm leaving this alone for now except adding handling for the quote.